internal/controller/driver_controller.go

```diff
@@ -327,6 +328,63 @@ func (r *driverReconcile) LoadAndValidateDesiredState() error {
 	return nil
 }
 
+func (r *driverReconcile) reconcileLogRotateConfigMap() error {
+	logRotateConfigmap := &corev1.ConfigMap{}
+	logRotateConfigmap.Name = utils.LogRotateConfigMapName(r.driver.Name)
+	logRotateConfigmap.Namespace = r.driver.Namespace
+
+	log := r.log.WithValues("logRotateConfigMap", logRotateConfigmap.Name)
+	log.Info("Reconciling logRotate configmap")
+
+	logRotationSpec := cmp.Or(r.driver.Spec.Log, &csiv1a1.LogSpec{}).Rotation
+	if logRotationSpec != nil {
+		opResult, err := ctrlutil.CreateOrUpdate(r.ctx, r.Client, logRotateConfigmap, func() error {
+			if _, err := utils.ToggleOwnerReference(true, logRotateConfigmap, &r.driver, r.Scheme); err != nil {
+				log.Error(err, "Failed adding an owner reference on the LogRotate config map")
+				return err
+			}
+			if logRotationSpec.Periodicity != "" && logRotationSpec.MaxLogSize.String() == "0" {
```

Comment:
```suggestion
if logRotationSpec.Periodicity != "" && logRotationSpec.MaxLogSize.IsZero() {
```

use helper function IsZero to check if it's zero.